packages/quantic/force-app/main/default/lwc/quanticRecentQueries/quanticRecentQueries.css

```diff
@@ -0,0 +1,20 @@
+.container {
+  width: 100%;
+  margin: 0 2em 0 0;
+}
```

Comment:
you might want to add this as well so the onclick is triggered properly when the click is on the text or icon.

```
li span {
  pointer-events: none;
}
```